documents/src/pages/elements/number-field.md

```diff
@@ -155,7 +156,11 @@ You can add the event listener `error-changed` to the element and it will dispat
 ::number-field::
 const element = document.getElementById('input');
 const errorChangedText = document.getElementById('error-text');
-element.value = "-1"
+element.value = "-1";
+
+setTimeout(() => {
+  element.reportValidity();
+}, 1000);
```

Comment:
I think setTimeout 1000ms is a bit unreal (too much) if users really need to verify the input value? do you have any smooth approach to call reportValidity()? or just use 50ms instead.

Reply:
Actually, this code doesn't make sense to show in docs. I'd prefer to just mention about `input.reportValidity()`. Do we really need to show this live sample?

Reply:
I agree to not add reportValidity() to the live sample. 
I talked with Nut that setTimeout to call reportValidity() is so weird either, users may usually call reportValidity() inside their app lifecycle.

Reply:
In my opinion, it would be good if we can show the users how to apply `input.reportValidity()` during initialization. However, after I reconsider about this again, it would be better not to include it in the docs because we have already mentioned it in the first paragraph of this topic. Thank you for your suggestion.

Reply:
@Nantawat-Poothong Yeah exactly. Also, we dont' need to have live demo in everywhere.